t/16-utils.t

```diff
@@ -502,4 +503,12 @@ subtest 'human readable size' => sub {
     is(human_readable_size(245760), '240 KiB', 'less than a MB');
 };
 
+subtest 'create downloads list' => sub {
+    is_deeply create_downloads_list({}), {}, 'empty for no settings provided';
+    is_deeply create_downloads_list({TEST_URL => 'test', TEST => 'test'}), {}, 'hash for download';
```

Comment:
In the test label I suggest to state the expectation, e.g. "empty for non-asset URL". The next test can then be "valid download for asset URL"